usersync/usersyncers/syncer.go

```diff
@@ -18,7 +18,8 @@ import (
 	"github.com/prebid/prebid-server/adapters/consumable"
 	"github.com/prebid/prebid-server/adapters/conversant"
 	"github.com/prebid/prebid-server/adapters/eplanning"
-	"github.com/prebid/prebid-server/adapters/grid"
+	"github.com/prebid/prebid-server/adapters/gamoshi"
+  "github.com/prebid/prebid-server/adapters/grid"
```

Comment:
There are now two identical import lines for gamoshi and the grid line isn't aligned like the others.  Can you please correct these two issues?